### PR TITLE
[feature] API docs theme #197

### LINKED ISSUE #197: [feature] API docs theme
The DRF browsable API and DRF Yasg theme we ship for REST APIs are quite different from one another and also different from the main theme of OpenWISP.

It would be great if we could make the DRF browsable API and DRF Yasg theme more similar to the admin theme by recyling the same CSS, so that just adding the openwisp theme css would be enough to make these look more similar.

@niteshsinha17 do you think this is doable? We could do it at the end of GSoC if there's still time.

### PR DESCRIPTION
**Pull request by Aryamanz29**
Closes #197

#### Changes

- Changed API docs theme to look similar to `admin`

#### `BEFORE`

Screenshot from 2022-02-06 22-52-44

Screenshot from 2022-02-11 16-04-32

#### `AFTER`

https://user-images.githubusercontent.com/56113566/152698653-19670156-ac52-4f53-be05-67bb95d96950.mp4

https://user-images.githubusercontent.com/56113566/153577528-ab644cc6-0daa-4208-92ce-97cb4c157170.mp4

#### Checklist

- [x] I have read the [contributing guidelines](http://openwisp.io/docs/developer/contributing.html#how-to-commit-your-changes-properly).
- [x] I have manually tested the proposed changes.
- [ ] I have written new test cases to avoid regressions. (if necessary)
- [ ] I have updated the documentation. (e.g. README.rst)
- [ ] I have added [change!] to commit title to indicate a backward incompatible change. (if required)
- [ ] I have checked the links added / modified in the documentation.

### CODE REVIEW

**Comment by Aryamanz29** [comment]
## `UPDATE :`

Screenshot from 2022-01-29 02-50-58

Screenshot from 2022-01-29 02-51-38

**Comment by Aryamanz29** [comment]
@nemesisdesign  Do I have to make changes in `OP block` and `Models` CSS as well to look similar to **admin-theme**? 😅

Screenshot from 2022-01-29 02-49-26

Screenshot from 2022-01-29 02-49-37

**Comment by nemesisdesign** [comment]
> @nemesisdesign Do I have to make changes in `OP block` styles as well to look similar to **admin-theme**? sweat_smile
> 
> Screenshot from 2022-01-29 03-44-55

No this one let's not touch it.

File: openwisp_utils/admin_theme/static/rest_framework/css/browsable-api.css [hunk 1]
```diff
@@ -11,3 +11,118 @@
...
+}
+::-webkit-scrollbar-thumb:hover {
+  background: #333;
+}
+.navbar {
+  background: #f9f9f9;
+  color: #777;
+  border: 0;
+}
+.navbar .nav li, .navbar .nav li a{
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/static/rest_framework/css/browsable-api.css:43]: some improvements we tried:

```css
.navbar .nav li, .navbar .nav li a {
    margin-top: 20px;
    color: #000;
    background-color: rgba(0, 0, 0, 0.1);
    font-weight: bold;
    padding: 0.4em 0.9em 0.5em;
    text-align: center;
    border-radius: 0.25em;
}
```

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Thank you very much for your great work, is looking really good, let's make some more improvements!

### Swagger UI

Can you please make the authorize button look like:

Screenshot from 2022-02-11 11-51-35

Also, would be nice to switch the color of links (`<a>` elements) to the same color we use in the openwisp admin, making sure the links are not bold.

Take a look also at these buttons please (which do not look right):
Screenshot from 2022-02-11 11-59-02

### DRF browsable API

Here it is what I suggest:

- remove the gird background
- make the header background white
- make the logo and spacing around it similar to what's in the swagger API
- the labels of the form I would try to remove the bold and make it full black (`#000`)
- the tabs in the form, let's not make the background reddish but keep the background consistent with the rest of the form box, the background color on hover could be slightly lighter (eg: `#f9f9f9`)

**Comment by Aryamanz29** [comment]
### Swagger UI

- [x] Screenshot from 2022-02-11 11-51-35

- [x] Also, would be nice to switch the color of links (`<a>` elements) to the same color we use in the openwisp admin, making sure the links are not bold.

####  NOW
Screenshot from 2022-02-12 15-06-45

- [x] Take a look also at these buttons please (which do not look right): Screenshot from 2022-02-11 11-59-02

#### NOW
Screenshot from 2022-02-12 15-02-23

 
### DRF browsable API

- [x]  remove the gird background
- [x]  make the header background white
- [x]  make the logo and spacing around it similar to what's in the swagger API
- [x]  the labels of the form I would try to remove the bold and make it full black (`#000`)
- [x]  the tabs in the form, let's not make the background reddish but keep the background consistent with the rest of the form box, the background color on hover could be slightly lighter (eg: `#f9f9f9`)

### NOW

Screenshot from 2022-02-12 15-09-12

Screenshot from 2022-02-12 15-09-23

**Review by nemesisdesign — APPROVED** [review]
Thanks :+1:

**Pull request merged**